inc/core/limited_offers.php

```diff
@@ -0,0 +1,336 @@
+			wp_die( 'Invalid nonce! Refresh the page and try again.' );
+		}
+
+		// We record the time and the plugin of the dismissed notification.
+		update_option( $this->wp_option_dismiss_notification_key_base . $this->active, 'neve_' . $this->active . '_' . current_time( 'Y_m_d' ) );
+		wp_die( 'success' );
+	}
+
+	/**
+	 * Render the dashboard banner.
+	 *
+	 * @return void
+	 */
+	public function render_notice() {
+
+		if ( ! $this->has_priority() ) {
+			return;
+		}
+
+		$message = 'Neve <strong>Black Friday Sale</strong> - Save big with a <strong>Lifetime License</strong> of Neve Agency Plan. <strong>Only 100 licenses</strong>, for a limited time!';
+
+		?>
+		<style>
+			.themeisle-sale {
+				padding: 10px 15px;
+
+				display: flex;
+				align-items: center;
+			}
+			.themeisle-sale svg {
+				margin-right: 15px;
+			}
+			.themeisle-sale a {
+				margin-left: 5px;
+			}
+			.themeisle-sale-error {
+				color: red;
+			}
+		</style>
+		<div class="themeisle-sale notice notice-info is-dismissible">
+			<div class="notice-dismiss"></div>
+			<svg width="24" height="24" viewBox="0 0 61 60" fill="none" xmlns="http://www.w3.org/2000/svg">
+				<path fill-rule="evenodd" clip-rule="evenodd" d="M0.5 0.198486H61V59.9128H0.5V0.198486ZM25.0305 30.0698V44.2415H17.7858V15.4382C17.7858 15.2989 17.8281 15.2013 17.9129 15.1456C17.9976 15.0899 18.1529 15.1595 18.3789 15.3546L36.4696 30.0698V15.8145H43.7143V44.6596C43.7143 44.8268 43.672 44.9313 43.5872 44.9731C43.5025 45.0149 43.3472 44.9383 43.1212 44.7432L25.0305 30.0698ZM43.7143 48.9127H17.7858V51.2699H43.7143V48.9127Z" fill="#0073AA"/>
+			</svg>
+
+			<span>
+				<?php echo wp_kses_post( $message ); ?>
+				<a href="<?php echo esc_url( ! empty( $this->offer_metadata['linkGlobal'] ) ? $this->offer_metadata['linkGlobal'] : '' ); ?>" target="_blank" rel="external noreferrer noopener">
+					Learn more
+				</a>
+			</span>
+			<span class="themeisle-sale-error"></span>
+		</div>
+		<script type="text/javascript">
+			window.document.addEventListener( 'DOMContentLoaded', () => {
+				const button = document.querySelector( '.themeisle-sale.notice .notice-dismiss' );
+				button?.addEventListener( 'click', e => {
+					e.preventDefault();
+					fetch('<?php echo esc_url( admin_url( 'admin-ajax.php' ) ); ?>', {
+						method: 'POST',
+						headers: {
+							'Content-Type': 'application/x-www-form-urlencoded'
+						},
+						body: new URLSearchParams({
+							action: 'dismiss_themeisle_event_notice_neve',
+							nonce: '<?php echo esc_attr( wp_create_nonce( 'dismiss_themeisle_event_notice_neve' ) ); ?>'
+						})
+					})
+						.then(response => response.text())
+						.then(response => {
+							if ( ! response?.includes( 'success' ) ) {
+								document.querySelector( '.themeisle-sale-error' ).innerHTML = response;
+								return;
+							}
+
+							jQuery( '.themeisle-sale.notice' ).fadeOut()
```

Comment:
Let's remove the use of jQuery. We can replace this with something like this:

```document.querySelector('.themeisle-sale.notice').classList.add("hidden");```

And in CSS:
```
.themeisle-sale.notice{
  opacity: 1;
  transition: opacity 2s;
}

.themeisle-sale.notice.hidden {
  opacity: 0;
}
```

Reply:
Thanks. I re-wrote it with the equivalent vanilla js.